package-lock.json

```diff
@@ -1,7697 +1,62 @@
 {
   "name": "@builder.io/html-to-figma",
   "version": "0.0.4",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Looks like you ran `npm install` with an older version of node. Do you mind re-running with `v16`, the one in `.nvmrc`?

Reply:
Done